runtime/compiler/control/J9Options.cpp

```diff
@@ -1685,12 +1686,17 @@ J9::Options::fePreProcess(void * base)
             j9nls_printf(PORTLIB, J9NLS_INFO, J9NLS_JIT_OPTIONS_XLP_EXTRA_COMMA);
          }
       // Parse Size -Xlp<size>
-      else if (xlpIndex >= 0)
+      else if (xlpIndex > largePageArgIndex)
          {
          // GET_MEMORY_VALUE macro casts it's second parameter to (char**)&, so a pointer to the option string is passed rather than the string literal.
          char *lpOption = "-Xlp";
          GET_MEMORY_VALUE(xlpIndex, lpOption, requestedLargeCodePageSize);
          }
+      // Parse -XX:LargePageSizeInBytes=<size>
+      else if ((largePageArgIndex >= 0) && (vm->largePageSizeRequested != -1))
```

Comment:
would it be better `(-1 != largePageArgIndex)` ?

Reply:
Same applies here.  https://github.com/eclipse/openj9/pull/7476#discussion_r389852533